includes/RestApi/JetpackController.php

```diff
@@ -82,7 +99,10 @@ public function set_options( $request ) {
 				);
 			}
 
-			if ( 'critical-css-premium' === $field['id'] ) {
+			if ( 'critical-css' === $field['id'] && 1 === (int) $field['value'] ) {
+				$css = new Regenerate_CSS();
```

Comment:
Should we check if the class exists just in case the user deactivated the plugin in another tab but still clicked on our UI?